jekyll/_cci2/circleci-images.md

```diff
@@ -93,37 +106,93 @@ The following packages are installed with `apt-get` on every image:
 - [xvfb](https://packages.debian.org/stretch/xvfb)
 - [zip](https://packages.debian.org/stretch/zip)
 
-### Other Packages
-
 The following packages are installed via `curl` or other means:
 
 - [Docker client](https://docs.docker.com/install/)
 - [Docker Compose](https://docs.docker.com/compose/overview/)
 - [dockerize](https://github.com/jwilder/dockerize)
 - [jq](https://stedolan.github.io/jq/)
 
-## Image Variants
+## Customizing a Convenience Image
+
+You can customize convenience images by adding image tags.
```

Comment:
I have a few thoughts here.

1. Using an existing image tag isn't 'customizing' an image. It's a common use case for images. Technically, a tag is always used (as `latest` will be used if left out). Most people use a tag.
2. The number 1 reason for using a tag, and ideally the most specific tag possible, is to achieve [Deterministic Builds](https://circleci.com/blog/preserve-build-integrity-prevent-problems-deterministic-builds/).
3. Using a SHA hash, if you're referring to the image digest, it's a tag. The digest ID is considered something different. https://docs.docker.com/engine/reference/commandline/pull/
4. The last option, "You want to modify an image by installing additional tools
+or changing the behavior of the image." would be done by extending an image.